### ISSUE TITLE
Drag & Drop functionality not working, image not shown

### ISSUE THREAD

**Issue by nurmuhammet-ali** [shopperlabs/framework#67]
Hello,

On brands and categories, Drag & Drop functionality not working, image is not showing

I made a small video: https://nurmuhammet.com/Record.mp4

**Comment by mckenziearts** [comment]
Hi
Yes, this feature was still not available. But in the new release ([v0.0.6](https://github.com/shopperlabs/framework/releases/tag/v0.0.6)) I replaced the management of files by the Spatie package `media-library` ([link here](https://spatie.be/docs/laravel-medialibrary/v9/introduction)) and with the use of Filepond which allows drag n drop

**Comment by nurmuhammet-ali** [comment]
@mckenziearts So, do you mean it is available now?

**Comment by mckenziearts** [comment]
Yes it's available I just published a draft. But the files are now managed by Laravel Media library, so you have to upload again the images for the products, brands, categories and collections so that they are visible

**Comment by nurmuhammet-ali** [comment]
@mckenziearts Well, ok, thank you. 
The overall project code is really well written.
I forked it and changed the lines to better implement slugs and more. Thanks, it helps a lot.

**Comment by mckenziearts** [comment]
Update `shopper/framework` to `^0.0.6`.  run `php artisan vendor:publish` to publish spatie/laravel-medialibrary migrations and `php artisan migrate`
